lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -59,7 +59,7 @@ def investigate(processed_source)
         private
 
         KEYWORDS = %w[begin class def end module].freeze
-        ALLOWED_COMMENTS = %w[:nodoc: rubocop:disable].freeze
+        ALLOWED_COMMENTS = %w[:nodoc: :yields: rubocop:disable].freeze
```

Comment:
This change also update the document.
https://github.com/rubocop-hq/rubocop/blob/v0.63.1/lib/rubocop/cop/style/commented_keyword.rb#L9

And it may be better to be able to customize with default.yml (.rubocop.yml) in the future :-)

Reply:
The document says “such as `:nodoc:` and `rubocop:disable`”, which I thought means it doesn’t need to list *all* allowed comments. Would you prefer listing all allowed comments there? E.g.
```
# Note that some comments (`:nodoc:`, `:yields:, and `rubocop:disable`)
# are allowed.
```
I wouldn’t go for full customization (giving an array in default.yml) just yet. I’d be interested to see – when a bug is reported – which kind of comment would need to be added to the list.

Reply:
> Would you prefer listing all allowed comments there? E.g.

Yes. I think that's fine.

> when a bug is reported – which kind of comment would need to be added to the list.

OK. Let's think about it at that time.

Reply:
I’ve pushed the new comments.

Reply:
Thanks for quick updating!